workflow.md

```diff
@@ -56,9 +57,18 @@ To contribute to this project, you need to use the JOSM editor.  Here are some r
 * If you don't see both data sources (drcog...osm & Data Layer x), make sure your JOSM remote control is enabled, restart JOSM and try again from the selecting **Edit in JOSM** from the Tasking Manager step above. 
 * You may also have an imagery layer as some projects may have a preferred imagery to use for comparison that will automatically load. If not, bring in Bing and any others you prefer; most likely existing data will be mapped/aligned to Bing. Ideally align to available GPS traces; where there is none or still is questionable, align imagery and DRCOG data to existing mapping or Bing where there is little to no existing data.
 
+### Pre-checking the data before merging
+
+The overall workflow is to "merge" the DRCOG data into your OSM Data Layer, while reviewing both data layers for possible conflicts. Some quality checks before merging will make things easier.
+
+* With the .osm file layer activated (no other OSM data visible), run the **JOSM Validator** (SHIFT+V) to look for errors. Common errors at this stage can include multiple buildings with the same address (usually an error where a house and a garage or shed have been given the same address tags). Refer to the section below on **fixing common errors** for solutions and guidelines.
+* While still in the new .osm file layer, **Select All** (CTRL+A) to select all objects, and then **Deselect Nodes** (SHIFT+U) to select all of the buildings. In the Selection window on the right side of the screen, scroll through the list of selected buildings to look for malformed street names. Examples of bad street names include:
+ * unexpanded abbreviations (Ct instead of Court, or Ln instead of Lane)
+ * use of the directional prefix "North" (not used in City and County of Denver)
```

Comment:
(not used in City and County of Denver *yet* so they aren't reflected on the street signs)